pkg/xds/generator/testdata/admin/07.envoy-config.golden.yaml

```diff
@@ -0,0 +1,115 @@
+resources:
+- name: kuma:envoy:admin
+  resource:
+    '@type': type.googleapis.com/envoy.config.cluster.v3.Cluster
+    altStatName: kuma_envoy_admin
+    connectTimeout: 5s
+    loadAssignment:
+      clusterName: kuma:envoy:admin
+      endpoints:
+      - lbEndpoints:
+        - endpoint:
+            address:
+              socketAddress:
+                address: ::1
+                portValue: 9901
+    name: kuma:envoy:admin
+    type: STATIC
+- name: kuma:readiness
+  resource:
+    '@type': type.googleapis.com/envoy.config.cluster.v3.Cluster
+    altStatName: kuma_readiness
+    connectTimeout: 5s
+    loadAssignment:
+      clusterName: kuma:readiness
+      endpoints:
+      - lbEndpoints:
+        - endpoint:
+            address:
+              socketAddress:
+                address: 127.0.0.1
```

Comment:
I think the address needs to be configurable because it might not work for ipv6. Lets verify it

Reply:
We don't support "IPv6 only" as of now, so either IPv4 or DualStack. So both of them include ipv4 address.

Reply:
It's supported now, so resolving.